lib/AccessSystem/TelegramBot.pm

```diff
@@ -570,7 +570,8 @@ sub induct_member ($self, $text, $message, $args = undef) {
     if ($tool && $person) {
         ## we're done here, actually try and do the induction
         # should we do this check before the find_person loop?
-        if (!$member->allowed->find({ tool_id => $tool->id, is_admin => 1})) {
+        my $allowed = $member->allowed->find({ tool_id => $tool->id });
+        if (!$allowed || !$allowed->is_admin) {
```

Comment:
It would be better to give a more specific reason. i.e. If you're not inducted, say that, if you're not an inductor, say that instead